scripts/variantstore/wdl/GvsExtractCallset.wdl

```diff
@@ -116,12 +116,16 @@ workflow GvsExtractCallset {
 
   # scatter for WGS and exome samples based on past successful runs and NOT optimized
   Int effective_scatter_count = if defined(scatter_count) then select_first([scatter_count])
-                                else if GetNumSamplesLoaded.num_samples < 100 then 100 # Quickstart
-                                   else if GetNumSamplesLoaded.num_samples < 1000 then 500
-                                      else if GetNumSamplesLoaded.num_samples < 5000 then 1000
-                                         else if GetNumSamplesLoaded.num_samples < 20000 then 2000 # Stroke Anderson
-                                            else if GetNumSamplesLoaded.num_samples < 50000 then 10000
-                                              else if is_wgs then 20000 else 7500
+                                else if is_wgs then
+                                     if GetNumSamplesLoaded.num_samples < 5000 then 1 # This results in 1 VCF per chromosome.
+                                     else if GetNumSamplesLoaded.num_samples < 20000 then 2000 # Stroke Anderson
```

Comment:
is there any reason to reference Stroke Anderson if we dont even have access to that data anymore?

Reply:
trying to manifest finding a copy of it somewhere? 🤷